gradlew.bat

```diff
@@ -64,29 +64,14 @@ echo location of your Java installation.
 
 goto fail
 
-:init
-@rem Get command-line arguments, handling Windows variants
-
-if not "%OS%" == "Windows_NT" goto win9xME_args
```

Comment:
I don't understand this. The goto doesn't have any effect, since the goto target is the next line anyway. That means that the labels and the goto is useless, right?

So if I understand correctly, we always did set `_SKIP=2`, and if `%~1` is empty, we would not pass any extra command line args to the Java invocation.

Both things are gone now. I suppose that means that we drop support for some very old Windows versions?

Reply:
> we always did set _SKIP=2

That's my understanding, too. I don't understand why the original author did this.

> if %~1 is empty, we would not pass any extra command line args to the Java invocation

If `%~1` is empty, that means user invoke `./gradlew.bat` without any parameters. In this case, it's same for `CMD_LINE_ARGS` to be empty or `%*` because `%*` is empty anyway.

> I suppose that means that we drop support for some very old Windows versions?

I don't see "we ever support very old windows versions", but anyway, it makes no sense to support a 20-years-ago-OS.

Reply:
Windows xp is not supported by Java 8, so it's totally safe to remove this code: https://www.oracle.com/java/technologies/javase/products-doc-jdk8-jre8-certconfig.html#os